Source/Control/adsr.h

```diff
@@ -66,11 +80,12 @@ class Adsr
     inline bool IsRunning() const { return mode_ != ADSR_SEG_IDLE; }
 
   private:
-    float   sus_, seg_time_[ADSR_SEG_LAST], a_, b_, y_, x_, prev_, atk_time_;
+    float   sus_level_,
+            seg_time_[ADSR_SEG_LAST]{0.f},
```

Comment:
Right now, seg_time_ has an entry for the sustain segment and the "idle" segment, simply because we use the enum to adress this array. Couldn't we just have three float values for the attack, decay and release stages and drop those two extra values?

Reply:
That's true. Idle and sustain entries are plain bogus and three floats would be even a bit faster ....

Reply:
Done.